spec/rubocop/cop/rails/root_pathname_methods_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Rails::RootPathnameMethods, :config do
+  {
+    Dir: described_class::DIR_METHODS,
```

Comment:
It's my personal preference, but I don't feel too confident when some collections from the code are used in tests directly. It may sound far-fetched, but what if someone accidentally removes a line from that array? Specs won't fail.
I'll leave this to @koic to decide on.